content/250-postgres/1100-integrations/500-vscode-agent.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'VS Code Agent' 
+description: 'Streamline Prisma ORM workflows, manage migrations, and create databases directly in Visual Studio Code using the Prisma Agent.' 
+metaTitle: 'Visual Studio Code Agent' 
```

Comment:
FWIW, I'd prefer the shorter spelling of "VS Code" instead of "Visual Studio Code" across the entire page but leave it up to you